.github/workflows/build_documentation.yml

```diff
@@ -1,6 +1,7 @@
 name: Build documentation
 
 on:
+  workflow_dispatch:
```

Comment:
If I understand correctly, we will have (at least) an input to allow specify the commit sha - as our goal is to build doc for some previous releases (tags).

cc @regisss too

Reply:
And we will have to change the part below

```
    with:
commit_sha: ${{ github.sha }}
```
to take into account of both case (push or workflow dispatch)